scripts/ci-build.sh

```diff
@@ -39,4 +39,4 @@ fi
 export PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig
 $EXEC pkg-config --cflags deltachat
 $EXEC pkg-config --libs deltachat
-$EXEC npm install --verbose;
```

Comment:
Isn't the full log useful on the CI server instead of having to hunt for it?  It shows exactly which compiler args were used etc.

Reply:
@flub yes it is but it was to long for travis. We didn't get a full log from travis, it always ended somewhere in the npm install. We should add that back on a proper ci :D